pages/irac-2022.md

```diff
@@ -49,13 +47,79 @@ We have invited renowned speakers who will give an introductory overview for eac
 ![picture](/assets/images/irac-2022-boya-goa.jpg){: style="height: 100px; width: 100px; border-radius: 100px; border: 3px solid #e9ecef;" .me-3 .float-start}
 **Boya Gao** - Boya is a post-doctor at plant protection college, [Nanjing Agricultural University](http://english.njau.edu.cn/mainm.htm) who researches insect migration and radar entomology.
 
-## Abstract submission
-
-[Abstract submission](https://forms.gle/8B9VVcf5Mq2tERhZ8){: .btn .btn-primary }
-
-<!-- ## Registration -->
+## Programme
+
+<style>
+  .table th:first-of-type {
+    width: 20%;
+  }
+</style>
+
+An optional pre-conference **bird watching excursion** to the Flüela Pass (with chance to see snowfinches) is organized by Fränzi Korner-Nievergelt. Email <irac2022@vogelwarte.ch> if you want to join.
+
+### Saturday - June 25
+
+time        | presentation
+----------- | ------------
+ 8:00-9:00  | Registration
+ 9:15-9:30  | Welcome
+**Session 1**{: .text-primary } | **Spatio-temporal movement patterns of birds, bats, and insects and their environmental influences**{: .text-primary }
+ 9:30-10:00 | **Liechti Felix** - Migration basics birds **keynote**{: .badge .bg-secondary }
```

Comment:
` Provide better titles for the introductory talks?`
I'd suggest to put them on TBD for now (to not further delay publication), and one of us sends an email to the keynote speakers to request their tentative titles.

Reply:
Thanks, see https://github.com/enram/globam.science/issues/37